vimrc

```diff
@@ -62,6 +62,7 @@ au BufRead,BufNewFile *.*.js.yug setfiletype javascript
 
 au BufRead,BufNewFile *.css.yug setfiletype css
 au BufRead,BufNewFile *.*.css.yug setfiletype css
+au BufReadPost *.scss set syntax=css
```

Comment:
Syntax incomplete. Insert BufRead, BufNewFile instead of BufReadPost.
For other info read this: http://vimdoc.sourceforge.net/htmldoc/autocmd.html